openmetadata-ui/src/main/resources/ui/src/components/ListView/ListView.component.tsx

```diff
@@ -59,20 +61,33 @@ export const ListView = <T extends object = any>({
         />
       </Col>
       <Col className="text-right" span={14}>
-        <Radio.Group
-          value={currentView}
-          onChange={(e) => setCurrentView(e.target.value)}>
-          <Radio.Button value={ListViewOptions.CARD}>
-            <Icon component={GridIcon} data-testid="grid" />
-          </Radio.Button>
-          <Radio.Button value={ListViewOptions.TABLE}>
-            <Icon component={ListIcon} data-testid="list" />
-          </Radio.Button>
-        </Radio.Group>
+        <Space align="center">
+          <span>
+            <Switch
+              checked={deleted}
+              data-testid="show-deleted-switch"
+              onChange={handleDeletedSwitchChange}
+            />
+            <Typography.Text className="m-l-xs">
+              {t('label.deleted')}
+            </Typography.Text>
+          </span>
```

Comment:
This should be a conditional render, It's common component so we can't keep adding things. We can make it based on props only